src/transforms/sort.jl

```diff
@@ -0,0 +1,52 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+
+"""
+    Sort(col, rev)
```

Comment:
```suggestion
    Sort(col; rev=false)
```
Only that. The rest of the code is perfect!